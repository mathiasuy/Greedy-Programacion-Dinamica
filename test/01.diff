1,2c1,21
< 1>Cantidad: 10
< 2>Fin.
---
> ==4594== Use of uninitialised value of size 8
> ==4594==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4594==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4594== 
> ==4594== Invalid write of size 8
> ==4594==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4594==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4594==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
> ==4594== 
> ==4594== 
> ==4594== Process terminating with default action of signal 11 (SIGSEGV)
> ==4594==  Access not within mapped region at address 0x0
> ==4594==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4594==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4594==  If you believe this happened as a result of a stack
> ==4594==  overflow in your program's main thread (unlikely but
> ==4594==  possible), you can try to increase the size of the
> ==4594==  main thread stack using the --main-stacksize= flag.
> ==4594==  The main thread stack size used in this run was 8388608.
> 1>timeout: the monitored command dumped core
> Segmentation fault (core dumped)
