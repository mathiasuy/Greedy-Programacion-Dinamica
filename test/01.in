max_cantidad_intervalos 
17
1-15-1
3-10-1
15-16-1
30-35-1
20-30-1
35-45-1
50-52-1
49-52-1
50-53-1
46-51-1
52-55-1
57-62-1
55-58-1
61-64-1
62-65-1
59-63-1
63-70-1
Fin
