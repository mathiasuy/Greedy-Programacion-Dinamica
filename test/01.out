1>Cantidad: 10
2>Fin.
